hypervisor/dm/vpci/vpci.c

```diff
@@ -138,13 +174,24 @@ void vpci_init(struct acrn_vm *vm)
 		.len = 4U
```

Comment:
@pre vm != NULL ?

Reply:
OK, will add, but looks misra c tool does not require this, so I do not add that